File: apiCampeonato/apiBackend/Models/ConfrontoModels.cs
namespace ApiBackend.Models;

public class Confronto
{
    public int ConfrontoId{ get; set; }
    public int TimeCasaId { get; set; }
    public int TimeForaId { get; set; }
    public Time TimeCasa { get; set; }
    public  Time TimeFora { get; set; }
    public Campeonato Campeonato { get; set; }
    public int CampeonatoId { get; set; }
    public int Gols_time_casa{ get; set; }
    public int Gols_time_fora{ get; set; }
}

File: apiCampeonato/FRONT/src/app/models/ConfrontoModels.ts
import { Time } from "@angular/common";
import { Campeonato } from "./CampeonatoModel";

export interface Confronto {
  ConfrontoId?: number;
  TimeCasaId: number;
  TimeCasa?: Time;
  TimeForaId: number;
  TimeFora?: Time;
  CampeonatoId: number;
  campeonato?: Campeonato;
  Gols_time_casa: number;
  Gols_time_fora: number;
}
